src/store/reducers/course.js

```diff
@@ -0,0 +1,30 @@
+import * as types from '../actions/actionTypes';
```

Comment:
That how store should looks like 
![image](https://user-images.githubusercontent.com/18098491/49308767-e1540900-f4e1-11e8-8cba-59aa1720792a.png)

Reply:
fixed